lms/djangoapps/teams/static/teams/js/views/team_discussion.js

```diff
@@ -13,6 +13,7 @@
                 render: function() {
                     var discussionModuleView = new DiscussionModuleView({
                         el: this.$el,
+                        readOnly: this.el.dataset.readOnly,
```

Comment:
I'd make this a boolean and not a string. So make this:

    readOnly: this.$el.data('read-only') === 'true'